plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/BuildPathTest.java

```diff
@@ -31,38 +32,53 @@
 import com.google.cloud.tools.eclipse.appengine.libraries.model.Library;
 
 public class BuildPathTest {
+  
+  private final List<Library> libraries = new ArrayList<>();
```

Comment:
No, that's JUnit. a new BuildPathTest object is created for each test. Only static variables are shared between tests.